src/core/trampoline.c

```diff
@@ -0,0 +1,57 @@
+#include <Python.h>
+#include <emscripten.h>
+
+// clang-format off
+EM_JS(
+PyObject*,
+py_emjs_trampoline_js,
+(PyCFunctionWithKeywords func, PyObject* self, PyObject* args, PyObject* kw),
+{
+    return wasmTableMirror[func](self, args, kw);
```

Comment:
Can't we use [`getWasmTableEntry`](https://github.com/emscripten-core/emscripten/blob/818523af5aacd41b8154906832a3c2b26e43a1b4/src/library.js#L3272-L3277) here? It also contains a fallback to wasmTable when wasmTableMirror is not available.